src/io.cr

```diff
@@ -908,19 +908,30 @@ abstract class IO
   # ```
   # io = IO::Memory.new("hello\nworld")
   # io.each_line do |line|
-  #   puts line.chomp.reverse
+  #   puts line
```

Comment:
The chomp maybe was there from times when the each_line didn't have a chomp option.

I think it's unclear what are the available options. We should add see `IO#gets` for available options.

Reply:
I'd like to change that in a different PR.